app/cli/internal/action/attestation_status.go

```diff
@@ -150,6 +136,78 @@ func (action *AttestationStatus) Run(ctx context.Context, attestationID string)
 	return res, nil
 }
 
+// populateMaterials populates the materials in the attestation result regardless of where they are defined
+// (contract schema or inline in the attestation)
+func populateMaterials(craftingState *v1.CraftingState, res *AttestationStatusResult) error {
+	visitedMaterials := make(map[string]struct{})
+	attsMaterials := craftingState.GetAttestation().GetMaterials()
+	inputSchemaMaterials := craftingState.GetInputSchema().GetMaterials()
+
+	if err := populateContractMaterials(inputSchemaMaterials, attsMaterials, res, visitedMaterials); err != nil {
+		return fmt.Errorf("adding materials from the contract: %w", err)
+	}
+
+	if err := populateAdditionalMaterials(attsMaterials, res, visitedMaterials); err != nil {
+		return fmt.Errorf("adding materials outside the contract: %w", err)
+	}
+
+	return nil
+}
+
+// populateContractMaterials populates the materials that are defined in the contract schema
+func populateContractMaterials(inputSchemaMaterials []*pbc.CraftingSchema_Material, attsMaterial map[string]*v1.Attestation_Material, res *AttestationStatusResult, visitedMaterials map[string]struct{}) error {
+	for _, m := range inputSchemaMaterials {
+		materialResult := &AttestationStatusResultMaterial{
+			Material: &Material{
+				Name: m.Name, Type: m.Type.String(),
+				Annotations: pbAnnotationsToAction(m.Annotations),
+			},
+			IsOutput: m.Output, Required: !m.Optional,
+		}
+
+		if cm, found := attsMaterial[m.Name]; found {
+			if err := setMaterialValue(cm, materialResult); err != nil {
+				return fmt.Errorf("setting material value: %w", err)
+			}
+		}
+
+		res.Materials = append(res.Materials, *materialResult)
+		visitedMaterials[m.Name] = struct{}{}
+	}
+	return nil
+}
+
+// populateAdditionalMaterials populates the materials that are not defined in the contract schema
+func populateAdditionalMaterials(attsMaterials map[string]*v1.Attestation_Material, res *AttestationStatusResult, visitedMaterials map[string]struct{}) error {
+	for name, m := range attsMaterials {
+		if _, found := visitedMaterials[name]; found {
+			continue
+		}
+
+		// No need to check for name collisions, as it is not defined in the contract schema and it's
+		// autogenerated by the crafter
+		materialResult := &AttestationStatusResultMaterial{
+			Material: &Material{
+				Name:        name,
+				Type:        m.GetMaterialType().String(),
+				Annotations: stateAnnotationToAction(m.Annotations),
+			},
+			// No need to check if the material is optional or not, as it is not defined in the contract schema
+			// TODO: Make IsOutput configurable
+			IsOutput: true, Required: false,
+		}
+
+		if err := setMaterialValue(m, materialResult); err != nil {
+			return fmt.Errorf("setting material value: %w", err)
+		}
+		materialResult.Set = true
```

Comment:
did you miss removing these lines from here?

Reply:
Yep my bad, removed.